packages/@okta/vuepress-theme-prose/assets/css/okta/icons/classes.scss

```diff
@@ -1693,11 +1693,66 @@ h1 {
   background-image: url('/img/icons/google-logo-32.png');
 }
 
+.icon.amazon::before {
+  content: '';
+  background-image: url('/img/icons/amazon-universal-gray-small.png');
+}
+
 .icon.azure::before {
   content: '';
   background-image: url('/img/icons/azure-ad-logo-75.png');
 }
 
+.icon.discord::before {
+  content:'';
+  background-image: url('/img/icons/discord.png');
+}
+
+.icon.github::before {
+  content:'';
+  background-image: url('/img/icons/github-universal-gray-small.png');
+}
+
+.icon.gitlab::before {
+  content:'';
+  background-image: url('/img/icons/gitlab.png');
+}
+
+.icon.line::before {
+  content:'';
+  background-image: url('/img/icons/line-20.png');
```

Comment:
are we not able to use the icons on the cards because they are too big?

Reply:
No, I believe the CSS code for stack selector icons reduces the image to 24x24. The problem with using the card icons is that some of them are SVG format. Another problem is that the card icons with the color black doesn't look good in dark mode (we'll have to fix this at some point). Alex Dahl explained to me that the older tooling for icons (okticon font) is not supported any more, and the only option to add icons is through the CSS method. I think this method only supports png or jpeg, not SVGs (or at least I haven't found a way to use SVGs yet).